src/event_imp.rs

```diff
@@ -804,29 +804,6 @@ impl ops::BitOr for Interests {
     }
 }
 
-impl ops::BitOrAssign for Interests {
```

Comment:
If we keep `BitOr` (which I think we are), we probably can keep `BitOrAssign` as well. WDYT?

Reply:
Sure I'll drop that commit.

Reply:
I've dropped the commit.